mne/source_space.py

```diff
@@ -1527,24 +1534,38 @@ def setup_volume_source_space(subject, fname=None, pos=5.0, mri=None,
             surf['rr'] += sphere[:3] / 1000.0  # move by center
             complete_surface_info(surf, do_neighbor_vert=True, copy=False)
         # Make the grid of sources in MRI space
-        sp = _make_volume_source_space(surf, pos, exclude, mindist, mri,
-                                       volume_label)
+        if volume_label is not None:
+            sp = []
+            for label in volume_label:
+                vol_sp = _make_volume_source_space(surf, pos, exclude, mindist,
+                                                   mri, label)
+                sp.append(vol_sp)
+        else:
+            sp = _make_volume_source_space(surf, pos, exclude, mindist, mri,
+                                           volume_label)
```

Comment:
in the sense I have to create the list of source space inside _make_volume_source_space?
Probably I didn't understand the change u asked me